composer.json

```diff
@@ -62,7 +62,7 @@
         "slickdeals/statsd": "3.1.0"
     },
     "require-dev": {
-        "appwrite/sdk-generator": "0.10.11",
+        "appwrite/sdk-generator": "dev-master",
```

Comment:
I would love us to create a new tag for this, give me a green light and I'll take care of it.

Reply:
Done :+1: